### ISSUE TITLE
View controllers don't have access to the global variables.

### ISSUE THREAD

**Issue by sansyrox** [sansyrox/robyn#447]
### Bug Description

Try creating a view and then accessing a global variable through that. It doesn't work.

Something like this doesn't work. Fix this. 
```
db = DB()
def AsyncView():
    async def get():
        res = db(bla)
        return "Hello, world!"
```

### Your operating system

None

### Your Python version (`python --version`)

None

### Your Robyn version

None

**Comment by Noborita9** [comment]
Does this have any news? I am really interested in working with Views. Any idea if making the Instance on the View does make it work?

**Comment by sansyrox** [comment]
@Noborita9 , apologies for the late revert. This comment missed my eye.

> Any idea if making the Instance on the View does make it work?

What do you mean by this?

**Comment by andryyy** [comment]
He probably means to create "db" from inside the view. But it doesn’t work either.

And functions that are not named by methods will raise an error iirc. Something along a wrong index value when nested functions are being collected.

**Comment by andryyy** [comment]
Perhaps it can be implemented with a decorator like @view_extension that can pass functions other than method handlers.

**Comment by Noborita9** [comment]
> He probably means to create "db" from inside the view. But it doesn’t work either.
Yeah exactly that

**Comment by sansyrox** [comment]
I don't think that will work. I am using this library for this - https://github.com/sansyrox/nestd/blob/main/nestd/__init__.py#L35

We will have to change something around this line imo.

**Comment by andryyy** [comment]
Yes, that’s where it throw the error. I have had no time to investigate further. :(

**Comment by sansyrox** [comment]
No worries. I will get around to it soon 😄

**Comment by sansyrox** [comment]
robyn doesn't have views.
